imports/components/cards/cards.Activity.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import { Component } from "react";
+import styled from "styled-components";
+import moment from "moment";
+import { Link } from "react-router";
+import { Error, Success } from "../icons";
+
+const Paragraph = styled.p`
```

Comment:
@richarddubay styled components are quite awesome! However, we can do all of these styles with junction 👍. I'd be happy to assist you in naming them!